Firmware/scripts/run_clangformat.py

```diff
@@ -0,0 +1,39 @@
+Import("env")
+import subprocess
+import os
+
+def linter_callback(*arg, **kwargs):
+    print("Executing Formatter")
+
+    check_folders = [env.get("PROJECT_INCLUDE_DIR"), env.get("PROJECT_SRC_DIR")]
+    
+    project_dir = env.get("PROJECT_DIR") + "\\"
+    check_folders_without_prefix = [str(i).replace( project_dir, '') for i in check_folders]
+
+    print("check folders", check_folders_without_prefix);
+    
+    subprocess.run("pip -q install clang-format")
```

Comment:
On macOS python installed via brew doesn't provide a pip binary only pip3, I swapped this to the following locally and it works as expected:
```
env.Execute("\"$PYTHONEXE\" -m pip install clang-format")
```

Reply:
Thanks, updated.

Reply:
With the current change I'm still getting failures during build:
```
FileNotFoundError: [Errno 2] No such file or directory: '"$PYTHONEXE" -m pip -q install clang-format':
```

Reply:
my bad, updated once more

Reply:
thanks working now